src/main/resources/application.properties

```diff
@@ -14,6 +14,8 @@ quarkus.datasource.username=hook
 quarkus.datasource.password=9FLK6cMm5px8vZ52
 quarkus.datasource.jdbc.url=jdbc:postgresql://192.168.1.139:5432/notifications
 quarkus.datasource.reactive.url=vertx-reactive:postgresql://192.168.1.139:5432/notifications
+quarkus.datasource.jdbc.max-size=4
+quarkus.datasource.reactive.max-size=95
```

Comment:
This parameter has no effect (the quarkus.datasource.reactive.* is used only by the Quarkus' forks of drivers)